sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/planetscale.yaml

```
@@ -11,10 +11,10 @@ provider:
       # - pscale_pw_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
       # - pscale_pw_0000000000000000000000000000000000000000000
       patternNot: 
-        "(?i)\
-        sample|\
-        example|\
-        (\\w)\\1{3,}"
+        - "(?i)\
+          sample|\
+          example|\
+          (\\w)\\1{3,}"
```

```suggestion
        # pscale_pw_example
        - "(?i)(s|ex)ample"
        # pscale_pw_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
        - "(\\w)\\1{3,}"
```

can you remove the passwords on top of this one? :pray: